packages/react-components/react-drawer/stories/src/Drawer/DrawerMultipleLevels.stories.tsx

```diff
@@ -50,54 +49,52 @@ const useStyles = makeStyles({
   },
 });
 
-const useMotionStyles = makeStyles({
-  toolbarButton: {
-    opacity: 0,
-    transform: 'translate3D(0, 0, 0) scale(0)',
-    transitionDuration: tokens.durationNormal,
-    transitionTimingFunction: tokens.curveEasyEase,
-    transitionProperty: 'opacity, transform',
-    willChange: 'opacity, transform',
-  },
-
-  toolbarButtonVisible: {
-    opacity: 1,
-    transform: 'translate3D(0, 0, 0) scale(1)',
-  },
-
-  level: {
-    opacity: 0,
-    transitionDuration: tokens.durationNormal,
-    transitionTimingFunction: tokens.curveEasyEase,
-    transitionProperty: 'opacity, transform',
-    willChange: 'opacity, transform',
-  },
-
-  levelVisible: {
-    opacity: 1,
-    transform: 'translate3D(0, 0, 0)',
-  },
-
-  level1: {
-    transform: 'translate3D(-100%, 0, 0)',
-  },
-
-  level2: {
-    transform: 'translate3D(100%, 0, 0)',
-  },
+const BodyPresenceMotion = createPresenceComponent<{ level: 1 | 2 }>(({ level }) => {
+  const keyframes = [
+    { opacity: 0, transform: level === 1 ? 'translateX(-100%)' : 'translateX(100%)' },
+    { opacity: 1, transform: 'translateX(0)' },
+  ];
+  const duration = motionTokens.durationNormal;
+  const easing = motionTokens.curveEasyEase;
+
+  return {
+    enter: {
+      keyframes: keyframes,
+      duration,
+      easing,
+    },
+    exit: {
+      keyframes: [...keyframes].reverse(),
+      duration,
+      easing,
+    },
+  };
+});
+const IconPresenceMotion = createPresenceComponent(() => {
+  const keyframes = [
+    { opacity: 0, transform: 'scale(0)' },
+    { opacity: 1, transform: 'scale(1)' },
+  ];
```

Comment:
This should be just `Scale` motion, but it does not support `from` yet. FYI @akatrukhin